jira/bin/komand_jira

```diff
@@ -28,11 +30,7 @@ def main():
     class ICONJira(insightconnect_plugin_runtime.Plugin):
         def __init__(self):
             super(self.__class__, self).__init__(
-                name=Name,
-                vendor=Vendor,
-                version=Version,
-                description=Description,
-                connection=connection.Connection()
+                name=Name, vendor=Vendor, version=Version, description=Description, connection=connection.Connection()
```

Comment:
I think RegenerationValidator fails due to these formatting changes here and in setup.py

Reply:
Ergh, probably because of Black